vpdq/vpdq-release.py

```diff
@@ -55,6 +56,7 @@ def run_command(command, cwd="."):
         subprocess.check_call(command, cwd=cwd)
     except subprocess.CalledProcessError as e:
         print(e.output)
+        sys.exit(1)
```

Comment:
I don't know why this check_call is caught. It means you can "install" the package, with no visible output, but it actually didn't install at all and will error if you do `python -c "import vpdq"`. Very confusing.

Reply:
I don't either...

Maybe trying to emulate a bash script without `-e` ? But that's not a great model to work off of. 

I wonder if you can just replace this entirely with subprocess directly at this point, I don't think there's any need for `run_command`